### PR TITLE
Add a SHiP test to verify `get_blocks_result_v1` works across Legacy and Savanna boundary

### PR DESCRIPTION
**Pull request by linh2931**
This test verifies SHiP `get_blocks_result_v1` works across Legacy and Savanna boundary.

In particular, the test sends a `get_blocks_result_v1` requesting blocks between block number 1 (pre Savanna) and a block whose block number greater than Savanna Genesis block (post Savanna). It then  verifies `finality_data` field in every block before Savanna Genesis block is NULL, `finality_data` field in every block after Savanna Genesis block contains a value.

Resolves https://github.com/AntelopeIO/spring/issues/111

### CODE REVIEW

**Review by heifner — CHANGES_REQUESTED** [review]

**Review by heifner — APPROVED** [review]

**Comment by ericpassmore** [comment]
Note:start
group: IF
category: TEST
summary: Ass Ship test to verify get_block_result_v1 works across both Legacy and Savanna finality modes.
Note:end

File: tests/ship_reqs_across_svnn_test.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                      specificExtraNodeosArgs=specificExtraNodeosArgs) is False:
+        Utils.cmdError("launcher")
+        Utils.errorExit("Failed to stand up cluster.")
+
+    # Verify nodes are in sync and advancing
+    cluster.waitOnClusterSync(blockAdvancing=5)
+    Print("Cluster in Sync")
+
+    #Verify nodes are in sync and advancing
+    cluster.waitOnClusterSync(blockAdvancing=3)
```
Comment (greg7mdp) [inline tests/ship_reqs_across_svnn_test.py:63]: Why call `waitOnClusterSync` twice?

File: tests/ship_reqs_across_svnn_test.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    Print("Shutdown unneeded bios node")
+    cluster.biosNode.kill(signal.SIGTERM)
+
+    shipNode = cluster.getNode(shipNodeNum)
+    # Block with start_block_num is before Savanna and block with end_block_num is after Savanna
+    start_block_num = 1
+    end_block_num = start_block_num + shipNode.getBlockNum()
```
Comment (greg7mdp) [inline tests/ship_reqs_across_svnn_test.py:70]: why `start_block_num + `?

**Review by greg7mdp — APPROVED** [review]

File: tests/ship_reqs_across_svnn_test.py (hunk 2 above)
Comment (linh2931) [reply tests/ship_reqs_across_svnn_test.py]: Thanks!

**Review by greg7mdp — APPROVED** [review]
